canonical-web-and-design/charmhub.io
# Fix a miss bit of blue

## Done
Updated the blue in the homepage hero strip

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- See that the hero strip has no blue
